MediaBrowser.Common/Extensions/HttpContextExtensions.cs

```diff
@@ -12,12 +12,18 @@ public static class HttpContextExtensions
         /// Checks the origin of the HTTP context.
         /// </summary>
         /// <param name="context">The incoming HTTP context.</param>
-        /// <returns><c>true</c> if the request is coming from LAN, <c>false</c> otherwise.</returns>
+        /// <returns><c>true</c> if the request is coming from the same machine, <c>false</c> otherwise.</returns>
```

Comment:
I also updated the docs to match the behaviour. The behaviour itself is unchanged from before this PR (other than working on UNIX socket when `X-Forwarded-For` is set).